integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/JakartaeeTomcatWithSecurityManagerIT.java

```diff
@@ -38,8 +37,9 @@ public JakartaeeTomcatWithSecurityManagerIT(final String tomcatVersion) {
     @Parameterized.Parameters(name = "Tomcat {0}")
     public static Iterable<Object[]> data() {
         return Arrays.asList(new Object[][]{
-            {"10.0.10-jdk11-adoptopenjdk-hotspot"},
-            {"10.1.0-jdk17-temurin"}, // Servlet 6.x
+            {"10.0.10-jdk11-adoptopenjdk-hotspot"}
+            // TODO: investigate why on 10.1.0-jdk17-temurin it returns 500 for the status page
+            //{"10.1.0-jdk17-temurin"}, // Servlet 6.x
```

Comment:
I've created an issue for this one:
https://github.com/elastic/apm-agent-java/issues/3953